modules/javafx.graphics/src/main/java/javafx/scene/text/TextFlow.java

```diff
@@ -490,23 +490,16 @@ public final DoubleProperty lineSpacingProperty() {
 
     /**
      * The size of a tab stop in spaces.
-     * Values less than 1 are treated as 1.
+     * Values less than 1 are treated as 1. This value overrides the
+     * {@code tabSize} of contained {@link javafx.scene.text.Text Text} nodes.
```

Comment:
`{@link Text}` is enough, but the FQN is also fine.

Does `Text`'s `tabSize` need a note that its value is overriden if the text is contained in a `TextFlow`?

Reply:
TextFlow isn't mentioned in the JavaDoc for any of the other Text properties where the same rule applies.  Perhaps that should be remedied with a follow-up task?

Reply:
A follow-up issue would be fine.

Reply:
I've created https://bugs.openjdk.java.net/browse/JDK-8235958 as a follow-up javadoc enhancement.